pom.xml

```diff
@@ -363,17 +363,17 @@
         <dependency>
             <groupId>org.apache.poi</groupId>
             <artifactId>poi</artifactId>
-            <version>4.0.0</version>
+            <version>4.1.1</version>
         </dependency>
         <dependency>
             <groupId>org.apache.poi</groupId>
             <artifactId>poi-ooxml</artifactId>
-            <version>4.0.0</version>
+            <version>4.1.1</version>
         </dependency>
         <dependency>
             <groupId>org.apache.poi</groupId>
-            <artifactId>poi-examples</artifactId>
-            <version>4.0.0</version>
+            <artifactId>poi-scratchpad</artifactId>
+            <version>4.1.1</version>
```

Comment:
I guess my only thought is that we could factor out all these "4.1.1" into a property like we do with `<junit.version>4.12</junit.version>` but that's just an optimization.